### PR TITLE
add questions.md

### PR DESCRIPTION
**Pull request by g-sathish**
Excited to learn github essentials already!!!

I request @mozzadrella to review the questions in the questions.md

### CODE REVIEW

**Comment by g-sathish** [comment]
Hey @mozzadrella!!!

File: questions.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Questions regarding pull requests
+
+    * Do we have to always create a pull request on a branch ?
```
Comment (mozzadrella) [inline questions.md:3]: Yes. And 99% of the time you are working on a branch anyway (master is a branch too!)

File: questions.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+    * Can we do some action based on the feedback from the pull requests ?
```
Comment (mozzadrella) [inline questions.md:4]: You can alter and update your code, push to the branch of your PR, and ask for additional feedback from there.

**Review by mozzadrella — APPROVED** [review]
Let me know if you have additional questions!
Looks good to me!

**Comment by g-sathish** [comment]
Hey @mozzadrella!!! Thank you very much for your answers.

**Pull request closed without merging**